indico/modules/events/cloning.py

```diff
@@ -173,13 +173,13 @@ def is_available(self):
         """
         return True
 
-    def has_conflicts(self, target_event):
-        """Check for conflicts between source event and target event.
+    def get_conflicts(self, target_event):
+        """Return conflicts between source event and target event.
 
         Use this when cloning into an existing event to disable options
         where ``target_event`` data would conflict with cloned data.
         """
-        return True
+        return []
```

Comment:
The original implementation returns `True` by default. Should we add a dummy conflict message just in case? Although, I don't see any place where this could cause problems..